homeassistant/scripts/commandline.py

```diff
@@ -0,0 +1,63 @@
+"""Script to retrieve information about a Home Assistant instance."""
+
+import argparse
+import pprint
+
+import homeassistant.remote as remote
+
+def run(args):
+    """The actual script body."""
+    # pylint: disable=too-many-locals,invalid-name,too-many-statements
+    parser = argparse.ArgumentParser(
+        description="Command-line interface for a Home Assistant instance")
+    parser.add_argument(
+        '-i', '--info',
+        action='store_true',
+        default=False,
+        help="show information about a Home Assistant instance")
+    parser.add_argument(
+        '-s', '--services',
+        action='store_true',
+        default=False,
+        help="get all available services.")
+    parser.add_argument(
+        '-e', '--entities',
+        action='store_true',
+        default=False,
+        help="get all entity states.")
+    parser.add_argument(
+        '-p', '--password',
+        metavar='api_password',
+        required=True,
+        help="the API password for the Home Assistant instance")
+    parser.add_argument(
+        '-r', '--remote',
```

Comment:
Because `-h` is used for help in many tools :wink: . But without abbreviation `--host` is the way to go, agreed.